common/hashes/hashes.go

```diff
@@ -62,3 +65,21 @@ func Simhash(data []byte) string {
 	hash := simhash.Simhash(simhash.NewWordFeatureSet(data))
 	return fmt.Sprintf("%d", hash)
 }
+
+func Jarm(host string) string {
+	t := target{}
+	if u, err := url.Parse(host); err == nil {
+		t.Host = u.Hostname()
+		port, _ := strconv.Atoi(u.Port())
+		t.Port = port
+	}
+	if t.Port == 0 {
+		t.Port = defaultPort
+	}
+	hash := fingerprint(t)
+	rexp, _ := regexp.Compile("(?m)0{62}")
```

Comment:
as the regexp is used many times we can move its declaration + compile at local package level:

```go
package xxx

var jarmHashRegex := regexp.MustCompile("(?m)0{62}")
```